<ui version="4.0" >
 <class>ConnectToServerDialog</class>
 <widget class="QDialog" name="ConnectToServerDialog" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>350</width>
    <height>150</height>
   </rect>
  </property>
  <property name="sizePolicy" >
   <sizepolicy>
    <hsizetype>5</hsizetype>
    <vsizetype>0</vsizetype>
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle" >
   <string>Connect To Server</string>
  </property>
  <property name="sizeGripEnabled" >
   <bool>false</bool>
  </property>
  <property name="modal" >
   <bool>true</bool>
  </property>
  <layout class="QVBoxLayout" >
   <property name="margin" >
    <number>9</number>
   </property>
   <property name="spacing" >
    <number>6</number>
   </property>
   <item>
    <layout class="QGridLayout" >
     <property name="margin" >
      <number>0</number>
     </property>
     <property name="spacing" >
      <number>6</number>
     </property>
     <item row="0" column="0" >
      <widget class="QLabel" name="serverLabel" >
       <property name="text" >
        <string>&amp;Server:</string>
       </property>
       <property name="buddy" >
        <cstring>serversHistoryCombo</cstring>
       </property>
      </widget>
     </item>
     <item row="0" column="1" >
      <widget class="KHistoryComboBox" name="serversHistoryCombo" >
       <property name="toolTip" >
        <string>Enter a server or select it from the dropdown list</string>
       </property>
      </widget>
     </item>
     <item row="1" column="0" >
      <widget class="QLabel" name="usernameLabel" >
       <property name="text" >
        <string>&amp;Username:</string>
       </property>
       <property name="buddy" >
        <cstring>usernameLine</cstring>
       </property>
      </widget>
     </item>
     <item row="1" column="1" >
      <widget class="KLineEdit" name="usernameLine" >
       <property name="toolTip" >
        <string>Enter your username</string>
       </property>
      </widget>
     </item>
     <item row="2" column="0" >
      <widget class="QLabel" name="passwordLabel" >
       <property name="text" >
        <string>&amp;Password:</string>
       </property>
       <property name="buddy" >
        <cstring>passwordLine</cstring>
       </property>
      </widget>
     </item>
     <item row="2" column="1" >
      <widget class="KLineEdit" name="passwordLine" >
       <property name="toolTip" >
        <string>Enter your password</string>
       </property>
       <property name="echoMode" >
        <enum>QLineEdit::Password</enum>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" >
     <property name="margin" >
      <number>0</number>
     </property>
     <property name="spacing" >
      <number>6</number>
     </property>
     <item>
      <widget class="QPushButton" name="editServersListButton" >
       <property name="toolTip" >
        <string>Click to edit the list of servers</string>
       </property>
       <property name="text" >
        <string>&amp;Edit Servers List...</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer>
       <property name="orientation" >
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" >
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="cancelButton" >
       <property name="toolTip" >
        <string>Click to cancel connecting to a server</string>
       </property>
       <property name="text" >
        <string>Cancel</string>
       </property>
       <property name="default" >
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="connectButton" >
       <property name="enabled" >
        <bool>false</bool>
       </property>
       <property name="toolTip" >
        <string>Click to connect to the server</string>
       </property>
       <property name="text" >
        <string>&amp;Connect</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>KHistoryComboBox</class>
   <extends>KComboBox</extends>
   <header>khistorycombobox.h</header>
  </customwidget>
  <customwidget>
   <class>KLineEdit</class>
   <extends>QLineEdit</extends>
   <header>klineedit.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
